README.md

```diff
@@ -42,55 +41,69 @@ Uptime card is highly customizable.
-| ok | color | #45C669 | v0.0.1 | Set the ok color.
-| ko | color | #C66445 | v0.0.1 | Set the ko color.
-| half | color | #C6B145 | v0.0.1 | Set the half color.
-| none | color | #C9C9C9 | v0.0.1 | Set the none color.
-| title | color | grey | v0.0.2 | Set the title text color, title_adaptive_color must be false.
-| status | color | gray | v0.0.2 | Set the status text color, status_adaptive_color must be false.
-| icon | color | | v0.0.2 | Set the icon text color, icon_adaptive_color must be false.
+| ok | color | `#45C669` | v0.0.1 | Set the `ok` color.
+| ko | color | `#C66445` | v0.0.1 | Set the `ko` color.
+| half | color | `#C6B145` | v0.0.1 | Set the half color.
+| none | color | `#C9C9C9` | v0.0.1 | Set the none color.
+| title | color | `grey` | v0.0.2 | Set the title text color, `title_adaptive_color` must be false.
+| status | color | `gray` | v0.0.2 | Set the status text color, `status_adaptive_color` must be false.
+| icon | color | | v0.0.2 | Set the icon text color, `icon_adaptive_color` must be false.
+| tooltip | color | grey | v0.1.0 | Set the tooltip text color, tooltip_adaptive_color must be false.
 
-### Show configuration
+### `Show` configuration
 
 **Parent key:** show
 
-| Name | Type | Default | Since | Description |
-|------|:----:|:-------:|:-----:|-------------|
-| header | boolean | true | v0.0.1 | Show the header.
-| icon | boolean | true | v0.0.1 | Show the icon.
-| status | boolean | true | v0.0.1 | Show the status.
-| timeline | boolean | true | v0.0.1 | Show the timeline.
-| footer | boolean | true | v0.0.1 | Show the footer.
-| average | boolean | true | v0.0.2 | Show the average.
+| Name | Default | Options | Description |
+|------:|:------:|:-----:|-------------|
+| header | `true` | `true`/`false` | Show/hide the header.
+| title | `true` | `true`/`false` | Show the title.
+| icon | `true` | `true`/`false` | Show/hide the icon.
+| status | `true` | `true`/`false` | Show/hide the status.
+| timeline | `true` | `true`/`false` | Show/hide the timeline.
+| average | `true` | `true`/`false` | Show/hide the average.
+| footer | `true` | `true`/`false` | Show/hide the footer.
 
 ### Alias configuration
 
 **Parent key:** alias
 
 | Name | Type | Default | Since | Description |
 |------|:----:|:-------:|:-----:|-------------|
-| ok | string | | v0.0.1 | Set a friendly name for ok state.
-| ko | string | | v0.0.1 | Set a friendly name for ko state.
+| ok | string | | v0.0.1 | Set a friendly name for `ok` state.
+| ko | string | | v0.0.1 | Set a friendly name for `ko` state.
+
+### Tooltip configuration
+
+**Parent key:** `tooltip`
+
+| Name | Type | Default | Since | Description |
+|------|:----:|:-------:|:-----:|-------------|
+| hour24 | boolean | `false` | v0.1.0 | Set to true to display 24-hour time.
+| template | string | `[[ from_date ]] - [[ to_date ]] | [[ average ]]%` | v0.1.0 | Set the template for the tooltip **[for more information](https://github.com/dylandoamaral/uptime-card#templating)**.
```

Comment:
It consider the `|` from `[[ from_date ]] - [[ to_date ]] | [[ average ]]%` as a new column :/.

Reply:
Yes I noticed that in Code preview. Will be resolved by the time this PR is ready.